hw/ip/otbn/doc/_index.md

```diff
@@ -496,7 +496,7 @@ This way, no alert is generated without setting an error code somewhere.
 ### Alerts
 
 OTBN has two alerts, one recoverable and one fatal.
-The [Error Conditions]({{< relref "hw/ip/otbn/doc#error-conditions" >}}) section has a detailed list of error conditions and the alerts that they raise.
+The {{< regref "ERR_CODE" >}} register documentation has a detailed list of error conditions and the alerts that they raise.
```

Comment:
I don't see any notion of alerts in the error description table (neither now nor before). Maybe just remove the second part of the sentence?